crt_portal/cts_forms/views.py

```diff
@@ -9,9 +9,27 @@
 
 
 class CRTReportWizard(SessionWizardView):
-    """Once all the sub-forms are submitted this class will clean data and save."""
     template_name = 'forms/report.html'
 
+    """Once all the sub-forms are submitted this class will clean data and save."""
```

Comment:
if we want to use http://www.sphinx-doc.org or some other automated doc generation in the future I think this docstring needs to be right under the class. Though, correct me if I am wrong.

Reply:
I'm sure you are right, feel free to move the docstring if you think that makes sense.